### DISCUSSION TITLE
how to set defult number in getPagerOptions in grid

### DISCUSSION THREAD

**Discussion by amirkian** [serenity-is/Serenity#5308] [Help]
I added this lines of code to my grid 
```
       protected getPagerOptions() {
            let opt = super.getPagerOptions();
            //Options in the dropdown
            opt.rowsPerPageOptions = [25];
  
 
            return opt;
        }

```
![image](https://user-images.githubusercontent.com/19525693/102190948-f257f300-3ecd-11eb-848f-5dc5f0e531f0.png)

as you can see the dropdown hasn't any default value.How can I set default value no to be show empty in the dropdown

**Comment by minhhungit** [comment]
hi @amirkian as I know at this time there are only 4 values you can use [20, 100, 500, 2000]
